features/claims/litigator/litigator_trial_claim_draft_submit.feature

```diff
@@ -33,11 +34,23 @@ Feature: Litigator partially fills out a draft final fee claim, then later edits
 
     Then I click "Continue" in the claim form
 
-    And I enter defendant, representation order and MAAT reference
-    And I add another defendant, representation order and MAAT reference
+    And I enter post agfs reform defendant, representation order and MAAT reference
```

Comment:
💭 Consider rewording to something like:
 
`And I enter the defendant details with a representation order date post AGFS reform`

This seems to describe better what we're filling, a _post agfs reform defendant_ seems a bit weird IMO.